ros_buildfarm/templates/snippet/rosdep_init.Dockerfile.em

```diff
@@ -0,0 +1,8 @@
+RUN mkdir -p /etc/ros/rosdep/sources.list.d
+
+@[for url in custom_rosdep_urls]@
+RUN (cd /etc/ros/rosdep/sources.list.d/ && wget --progress=dot @url )
+@[end for]@
+@[if not custom_rosdep_urls]@
+RUN rosdep init
```

Comment:
I think we need `rosdep init` here regardless of whether `custom_rosdep_urls` is set.